source/stream.c

```diff
@@ -32,7 +32,9 @@ int aws_input_stream_read(struct aws_input_stream *stream, struct aws_byte_buf *
     const size_t safe_buf_capacity = dest->capacity - dest->len;
     struct aws_byte_buf safe_buf = aws_byte_buf_from_empty_array(safe_buf_start, safe_buf_capacity);
 
+    __itt_task_begin(tracing_domain, __itt_null, __itt_null, tracing_stream_read_handle);
```

Comment:
very debatable: if most of these strings are only used once, it might be simpler to just declare them right where they're used? So you can just edit 1 file when you're inserting task markers, instead of 3? Like:

```suggestion
    static _itt_string_handle *tracing_stream_read_handle = __itt_string_handle_create("Stream::Read");
    __itt_task_begin(tracing_domain, __itt_null, __itt_null, tracing_stream_read_handle);
```